### PR TITLE
add sort logic to IdeaCollection process

### PR DESCRIPTION
**Pull request by der-john**

### CODE REVIEW

**Comment by slomo** [comment]
Works. A bit strange is that when I vote the sorting is update but the list item isn't (but this is unrelated).

File: src/meinberlin/meinberlin/static/js/Packages/Meinberlin/IdeaCollection/Process/Detail.html [hunk 1]
```diff
@@ -6,15 +6,17 @@
...
                     isBuergerhaushalt: isBuergerhaushalt,
                     isKiezkasse: isKiezkasse,
-                }">
+                }"
+                data-empty-text="{{ 'TR__PROPOSAL_EMPTY_TEXT' | translate }}"
+                data-ng-show="!isShowMap"
+                data-no-create-form="true"
+                data-path="{{path}}"
+                data-sort="sort"
+                data-sorts="sorts">
```
Comment (xi) [inline src/meinberlin/meinberlin/static/js/Packages/Meinberlin/IdeaCollection/Process/Detail.html]: These changes should also be applied to the map listing further down.

File: src/meinberlin/meinberlin/static/js/Packages/Meinberlin/IdeaCollection/Process/Process.ts [hunk 2]
```diff
@@ -34,11 +34,25 @@ export var detailDirective = (
         templateUrl: adhConfig.pkg_path + pkgLocation + "/Detail.html",
         scope: {
             path: "@",
+            sorts: "=?",
```
Comment (xi) [inline src/meinberlin/meinberlin/static/js/Packages/Meinberlin/IdeaCollection/Process/Process.ts]: You don't pass `sorts` as a parameter, so it should not be here.

Comment (xi) [inline src/meinberlin/meinberlin/static/js/Packages/Meinberlin/IdeaCollection/Process/Process.ts]: Fixed in #2341

File: src/meinberlin/meinberlin/static/js/Packages/Meinberlin/IdeaCollection/Process/Detail.html (hunk 1 above)
Comment (xi) [inline src/meinberlin/meinberlin/static/js/Packages/Meinberlin/IdeaCollection/Process/Detail.html]: Fixed in #2341

**Pull request merged**